oci/private/tarball_run.sh.tpl

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -o pipefail -o errexit -o nounset
+
+readonly IMAGE="{{image_path}}"
+if command -v docker &> /dev/null; then
+    docker load --input "$IMAGE"
+elif command -v podman &> /dev/null; then
+    podman load -i "$IMAGE"
+else
+    echo "Neither docker or podman could be found."
```

Comment:
this would be a nice place to point out the `run_template` attribute since it's the next step for a user who hits this